src/pages/plugins/PluginCardSkeleton.vue

```diff
@@ -38,9 +38,31 @@ export default {
 
 <style lang="scss">
 .k-skeleton-25 {
-  --KSkeletonCardWidth: 24%;
   .skeleton-card {
     height: 240px;
   }
+
+  // --KSkeletonCardWidth: 24%;
```

Comment:
Can we remove this line?

Reply:
Sure. This was intended to leave a trace of the equivalent variable definition in Kongponents.